dataduct/pipeline/utils.py

```diff
@@ -102,7 +106,8 @@ def list_pipeline_instances(pipeline_id, conn=None, increment=25):
         instances(list): list of pipeline instances
     """
     if conn is None:
-        conn = DataPipelineConnection()
+        region = next((x for x in regions() if x.name == str(REGION).lower()), None)
+        conn = DataPipelineConnection(region=region)
```

Comment:
can you create a utils function as `get_datapipeline_connection` and we can just use it at all three places instead of having the same thing repeated thrice.